packages/renderer/src/lib/ui/TreeView.svelte

```diff
@@ -0,0 +1,79 @@
+<script context="module" lang="ts">
+const _expansionState = new Map<string, boolean>();
+</script>
+
+<script lang="ts">
+import type { FileNode } from '../../../../main/src/plugin/file-tree';
+import { ImageUtils } from '../image/image-utils';
+
+export let tree: FileNode<any>;
+export let margin = 0;
```

Comment:
IMHO it can't be in ui folder as it's not a generic component

it includes ImageUtils and file-tree

Reply:
either it needs to be a fully generic `TreeView` or its location is in `renderer/src/lib/image` folder

Reply:
and to avoid confusion, it should be named ImageFileTreeView (or something like that)